packages/api/src/beacon/routes/lodestar.ts

```diff
@@ -125,6 +129,7 @@ export type Api = {
  */
 export const routesData: RoutesData<Api> = {
   writeHeapdump: {url: "/eth/v1/lodestar/writeheapdump", method: "POST"},
+  writeNetworkThreadProfile: {url: "/eth/v1/lodestar/writenetworkthreadprofile", method: "POST"},
```

Comment:
Every route uses different styles:
- `someroute`
- `some-route`
- `some_route` <- let's use this one here to align with beacon APIs https://github.com/ethereum/beacon-APIs/blob/master/beacon-node-oapi.yaml

```suggestion
  writeNetworkThreadProfile: {url: "/eth/v1/lodestar/write_network_thread_profile", method: "POST"},
```

I can PR for the rest after merging this PR